companies/ups-de.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "ups-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "United Parcel Service Deutschland S.à r.l. & Co. OHG",
+    "runs": [
+        "UPS Paketdienst",
+        "UPS My Choice"
+    ],
+    "address": "c/o UPS Europa SA\nAve Ariane 5\n1200 Brussel\nBelgium",
```

Comment:
CustomerServiceDEDE@ups.com is listed on https://www.ups.com/de/de/support/contact-us.page